prime-dotnet-webapi/Models/SelfDeclarationVersion.cs

```diff
@@ -17,5 +17,7 @@ public class SelfDeclarationVersion : BaseAuditable
         public string Text { get; set; }
 
         public DateTimeOffset EffectiveDate { get; set; }
+
```

Comment:
Needs a comment ... "A comma-separated list of all care setting codes for which applicable"?

Reply:
Good call, updated.